blocks.js

```diff
@@ -2518,6 +2547,14 @@ BlockMorph.prototype.userMenu = function () {
         'userDestroy'
     );
     menu.addItem(
+      'stringify...',
+        function() {
+            var code = myself.stringify();
+            window.prompt('scratchblocks code for you to copy and paste', code);
```

Comment:
It's mostly just layout / morphic code. 

And if we use it in 2 place, then we could create a `CopySelectionDialogMorph` and store it in widgets.js, so then it's like...no code. ;)